content/v2.markdown

```diff
@@ -129,3 +129,50 @@ For example:
     https://api.dnsimple.com/v2/_/domains?per_page=100&page=2
 
 
+## Rate limiting
+
+There is a limit to the number of requests that you can perform per hour.
+
+- For **authenticated requests** you can make **up to 4,000 requests per hour**.
+- For **unauthenticated requests**, you can make **up to 30 requests per hour**.
+
+Note that both authenticated and unauthenticated requests are associated with your IP address, and not the user or account making them.
```

Comment:
@jacegu I'm not sure I'd mention this as, especially for authenticated requests, we'll probably change over a token-based or account-based throttling at some point to avoid abuse or side effects of having client sending us requests from multiple IPs or the multiple clients from the same IP.

I propose to remove this line for now.